.github/workflows/build_hub.yml

```diff
@@ -91,15 +91,18 @@ jobs:
         run: |
           docker-compose -f docker-compose.yml up -d prime_dev redox ftps sftp azurite
           
-      - name: Wait for the API to become available
-        uses: cygnetdigital/wait_for_response@v2.0.0
+      - name: Give some time for the container to start
+        uses: juliangruber/sleep-action@v1
         with:
-          url: 'http://localhost:7071/api/lookuptables/list'
-          responseCode: '200,201,401'
-          timeout: 60000
-          interval: 5000
+          time: 30s
```

Comment:
Curious about the reason for the switch -- does the API availability method no longer work?

Reply:
We are having issues with making sure the tables and settings are loaded, so now we have a fool proof method to ensure that is loaded by using the reload commands.  When the API is available tends to be just before the tables and settings are loaded.